javascript/javascript1/week2/candyFunction.js

```diff
@@ -0,0 +1,64 @@
+let boughtCandyPrices=[];
+let price;
+const amountToSpend=Math.random() * 100;
+
+function addCandy(candyType, weight)
+{
+    
+    if(candyType=="Sweet")
+    {
+        price=0.5*weight;
+        boughtCandyPrices.push(price);
+    }
+    else if(candyType=="Chocolate")
+    {
+        price=0.7*weight;
+        boughtCandyPrices.push(price);
+    }
+    else if(candyType=="Toffee")
+    {
+        price=1.1*weight;
+        boughtCandyPrices.push(price);
+    }
+    else
+    {
+        candyType="Chewing Gum";
+        price=0.03*weight;
+        boughtCandyPrices.push(price);
+
+    }
+    return boughtCandyPrices.toString();
```

Comment:
I don't think calling `.toString()` is exactly what you'd want to do here to accomplish your task. You'll want the return type to remain an array so that you can loop over it for the `canBuyMoreCandy` function